src/main/java/com/example/springbootbookshop/mapper/BookMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.example.springbootbookshop.mapper;
+
+import com.example.springbootbookshop.dto.BookDto;
+import com.example.springbootbookshop.dto.CreateBookRequestDto;
+import com.example.springbootbookshop.entity.Book;
+import org.mapstruct.InjectionStrategy;
+import org.mapstruct.Mapper;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.NullValueCheckStrategy;
+
+@Mapper(componentModel = "spring",
+        injectionStrategy = InjectionStrategy.CONSTRUCTOR,
+        nullValueCheckStrategy = NullValueCheckStrategy.ALWAYS)
+public interface BookMapper {
+    BookDto toDto(Book book);
+
+    Book toBook(CreateBookRequestDto bookDto);
+
+    void updateBook(CreateBookRequestDto dto, @MappingTarget Book book);
```

Comment:
Let's avoid updating book values to null, when CreateBookRequestDto includes some null values

```suggestion
     @BeanMapping(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.IGNORE)
    void updateBook(CreateBookRequestDto dto, @MappingTarget Book book);
```